src/frontend/komponenter/Felleskomponenter/RegistrerDødsfallDato/RegistrerDødsfallDatoModal.tsx

```diff
@@ -23,30 +23,16 @@ interface IProps {
 }
 
 const RegistrerDødsfallDatoModal = ({ lukkModal, person, erLesevisning }: IProps) => {
-    const {
-        skjema,
-        valideringErOk,
-        registrerManuellDødsfall,
-        nullstillSkjema,
-        settVisfeilmeldinger,
-        settRestFeil,
-        restFeil,
-    } = useRegistrerDødsfallDatoSkjemaContext({
-        onSuccess: lukkModal,
-        person,
-    });
-
-    const lukkModalOgNullstillSkjema = () => {
-        nullstillSkjema();
-        settVisfeilmeldinger(false);
-        settRestFeil(undefined);
-        lukkModal();
-    };
```

Comment:
Når vi har flyttet logikken ut i egen fil for modalen er det ikke vits i å nullstille hver gang man lukker modalen. Logikken for modal er sånn at komponenten rendres på nytt hver gang man lukker modalen (siden koden ser sånn her ut: visModal && <ModalKomponent />), og da vil også dataen bli nullstilt/"starte på nytt" hver gang man åpner en modal.